prac_07/project.py

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Don't forget your program, class, and function doc-strings when you finish this practical.
I try to add blank ones as I go to remind myself.
`""""""`